dbt/scripts/utils/export.py

```diff
@@ -365,6 +366,8 @@ def save_model_to_workbook(
                             type_func = int
                         elif data_type == "float":
                             type_func = float
+                        elif data_type == "decimal":
+                            type_func = decimal.Decimal
```

Comment:
**question (non-blocking):** Just out of curiousity, what's the deal with the decimal type here? Does it maintain the fixed precision of the Parquet/spark types?

Reply:
Pretty much! We have to process a couple of fields that are stored as strings in iasWorld, but actually represent arbitrary numerics. The Python `decimal.Decimal` type should let us handle those columns gracefully, without the pitfalls of both `int` (which will truncate any decimals) and `float` (which will add decimals to ints). Annoying that we have to do this, but at least it works.